docs/basics/functions.md

```diff
@@ -832,13 +832,14 @@ Which would return the following:
 }]
 ```
 
-### split_array
+### split_array (leaf_array)
 
-| Applies to | Description                           |
-| :--------- | :------------------------------------ |
-| API        | Split an array that has nested arrays |
+| Applies to | Description |
+| :--------- | :---------- |
+| API | Split an array that has nested arrays |
```

Comment:
s/Split/Splits

@haihongren What does the function split the nested arrays into? New arrays at the root level of the split array?

Reply:
@theletterf given you have a simple array eg. [1,2,3,4] it would split that into 4 new samples.
Eg. sometimes APIs return an array of values, with another array of timestamps in the same payload. 

We need to split both of these out separately so the timestamp with its corresponding value can be maintained properly.

Reply:
@theletterf  Updated functions.md accordingly.